yarn.lock

```diff
@@ -1526,6 +1526,14 @@
   dependencies:
     moment "^2.24.0"
 
+"@dhis2/app-runtime-adapter-d2@^1.0.2":
+  version "1.0.2"
+  resolved "https://registry.yarnpkg.com/@dhis2/app-runtime-adapter-d2/-/app-runtime-adapter-d2-1.0.2.tgz#b1a88d371830bfe63fbc00068f7eda98f3c9259f"
+  integrity sha512-HbkkRdt4qMHAv3EKjZUuXXm7QfHK7NuxEHhxR33xxLfZHGdI+doRqvqzmpvJYkPDv4PaY91V4FKgnyznbDdlkQ==
+  dependencies:
+    "@dhis2/ui-core" "^4.6.1"
```

Comment:
Any way we can avoid having two versions of `@dhis2/ui-core`?

Reply:
Yes, the `@dhis2/app-runtime-adapter-d2` dependency had `@dhis2/ui-core` listed as dependency but was not used.
Now `@dhis2/app-runtime-adapter-d2` is updated.